management/src/test/java/org/kaazing/gateway/management/monitoring/configuration/impl/MMFMonitoringDataManagerTest.java

```diff
@@ -19,11 +19,13 @@
 
 import org.jmock.Expectations;
 import org.jmock.Mockery;
+import org.junit.Ignore;
 import org.junit.Test;
 import org.kaazing.gateway.management.monitoring.entity.manager.impl.ServiceCounterManagerImpl;
 import org.kaazing.gateway.management.monitoring.service.MonitoredService;
 import org.kaazing.gateway.service.MonitoringEntityFactory;
 
+@Ignore("doesn't work with the latest Agrona 0.4.12")
```

Comment:
@dpwspoon says that we haven't finished all the monitoring work using Agrona. So it is ok to ignore these tests.